framework/src/fviks/FVDiffusionInterface.C

```diff
@@ -49,12 +59,21 @@ FVDiffusionInterface::computeQpResidual()
                                           -one_over_gradient_support;
 
   ADReal diffusivity;
-  interpolate(Moose::FV::InterpMethod::Average,
-              diffusivity,
-              coef_elem(elemFromFace()),
-              coef_neighbor(neighborFromFace()),
-              *_face_info,
-              true);
+  if (_coeff_interp_method == Moose::FV::InterpMethod::Average)
+    interpolate(Moose::FV::InterpMethod::Average,
```

Comment:
why dont we want interpolate to forward to harmonic interpolation? 
This would simplify the code in the kernels

Reply:
Yeah, I tried it at the beginning (not the current implementation) and it looked like we instantiated an interpolate template which was outside the range of the harmonic interpolation with types that didn't have (1/something) defined. However, I am curious if the current implementation will exhibit this issue too, I'll check it real quick, having a one-liner would indeed be much nicer!